python/lsst/dax/imgserv/vo/dal/dal.py

```diff
@@ -163,3 +163,20 @@ def get_capabilities(self, params: dict) -> str:
         """
         resp = render_template("vosi_capabilities.xml")
         return resp
+
+    def get_jobs(self, params: dict) -> str:
+        """ Get the service capabilities.
+
+        Parameters
+        ----------
```

Comment:
Why even add this?  It's empty.  At least for your TODO put in a JIRA bug number and assign it to yourself to make sure it's not forgotten.

Also the text says service capabilities, not getting about jobs.  The params help text is also pretty useless, in that I can tell it's a dict, and it's parameters, but you're not saying what keys are useful, or a reference to some kind of standard format of these keys that are passed around.

Reply:
Have created a bunch of new JIRA tickets as followup to this work, and will paste those into the marked spots in the code.